src/DynamoCore/Nodes/Custom Nodes/CustomNodeController.cs

```diff
@@ -49,7 +49,7 @@ protected override AssociativeNode GetFunctionApplication(NodeModel model, List<
                 return AstFactory.BuildFunctionCall(Definition.FunctionName, inputAstNodes);
             }
 
-            var count = Definition.DisplayParameters.Count();
+            var count = Definition.DisplayParameters != null ? Definition.DisplayParameters.Count() : 0;
```

Comment:
This isn't your fault, but `Definition.DisplayParameters` should never be `null`. I have created [an internally tracked task](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6571) to set things right (you don't have to work on that, but if you wanted to, feel free).